packages/core-transaction-pool/__tests__/interface.test.js

```diff
@@ -185,6 +185,20 @@ describe('Transaction Pool Interface', () => {
     })
   })
 
+  describe('removeForgedTransactions', () => {
+    it('should be a function', () => {
+      expect(poolInterface.removeForgedTransactions).toBeFunction()
+    })
+
+    it('should throw an exception', async () => {
+      await expect(
+        poolInterface.removeForgedTransactions(),
+      ).rejects.toThrowError(
+        'Method [removeForgedTransactions] not implemented!',
+      )
```

Comment:
Use `expect(poolInterface.removeForgedTransactions).toThrow('Method [removeForgedTransactions] not implemented!')`.